Sources/tart/Commands/Run.swift

```diff
@@ -43,7 +43,16 @@ struct Run: AsyncParsableCommand {
     let task = Task {
       do {
         if let vncWrapper = vncWrapper {
-          await vncWrapper.open()
```

Comment:
Maybe we can keep the opening logic in the `VNCWrapper` (see https://github.com/cirruslabs/tart/pull/48#discussion_r902768135) and pass a boolean that instructs the method whether to open the URL instead?

Something like:

```swift
await vncWrapper.wait(open: !noGraphics && ProcessInfo.processInfo.environment["CI"] == nil)
```

Reply:
I though about it but then it looks weird to me. Plus the `!noGraphics && ProcessInfo.processInfo.environment["CI"] == nil` expression looks a bit hard to read.